js/src/views/Account/account.js

```diff
@@ -302,6 +324,63 @@ class Account extends Component {
     );
   }
 
+  renderExportDialog () {
+    const { toggleExportDialog } = this.store;
+    const { changePassword, accountValue, onExport } = this.exportStore;
+
+    if (!this.store.isExportVisible) {
+      return null;
+    }
+    return (
```

Comment:
Tiny style grumble, would prefer an empty line between blocks, just to aid readability somewhat.